### PR TITLE
Adding readme.java.md for agrifood

### PR DESCRIPTION
**Pull request by priyankajayaswal1**
@weidongxu-microsoft Please review readme for java.

### CODE REVIEW

File: specification/agrifood/resource-manager/readme.java.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+These settings apply only when `--java` is specified on the command line.
+Please also specify `--azure-libraries-for-java-folder=<path to the root directory of your azure-libraries-for-java clone>`.
+
+``` yaml $(java)
+java:
+  fluent: true
+  namespace: com.azure.agrifood
+  license-header: MICROSOFT_MIT_NO_CODEGEN
+  output-folder: $(azure-libraries-for-java-folder)/agrifood/arm-agrifood
```
Comment (weidongxu-microsoft) [inline specification/agrifood/resource-manager/readme.java.md:11]: You don't need any of them.

But where is the `title: AgriFoodMgmtClient`?

Comment (priyankajayaswal1) [reply specification/agrifood/resource-manager/readme.java.md]: updated.

Comment (weidongxu-microsoft) [reply specification/agrifood/resource-manager/readme.java.md]: Use `AgriFood` as prefix for entry of Java lib.

Use `AgriFoodManagementClient` (this is usually not visible to user) as Java does not prefer (inappropriately) abbreviated word ("Mgmt").

**Review by weidongxu-microsoft — APPROVED** [review]

**Pull request merged**